src/AstRunner/AstMap/ClassReferenceBuilder.php

```diff
@@ -188,4 +192,11 @@ public function constFetch(string $classLikeName, int $occursAtLine): self
 
         return $this;
     }
+
+    public function setInternal(bool $internal = true): self
```

Comment:
How about?
```suggestion
    public function markAsInternal(): self
```
I think there will be no use case for setting it to false afterwards